TwoFactorAuth/etc/di.xml

```diff
@@ -40,7 +40,6 @@
     <preference for="Magento\TwoFactorAuth\Api\DuoAuthenticateInterface" type="Magento\TwoFactorAuth\Model\Provider\Engine\DuoSecurity\Authenticate"/>
     <preference for="Magento\TwoFactorAuth\Api\Data\DuoDataInterface" type="Magento\TwoFactorAuth\Model\Data\Provider\Engine\DuoSecurity\Data"/>
     <preference for="Magento\TwoFactorAuth\Api\U2fKeyConfigReaderInterface" type="Magento\TwoFactorAuth\Model\Provider\Engine\U2fKey\ConfigReader"/>
-    <preference for="OTPHP\TOTPInterface" type="OTPHP\TOTP"/>
```

Comment:
Why was this removed?

Reply:
This class isn't within scope for dependency injection.

![Screenshot_2021-06-15_22-38-40](https://user-images.githubusercontent.com/334786/122126995-71e95f00-ce2a-11eb-976d-deec6563ba73.png)

From what I can tell, this is used as a return type in `Magento\TwoFactorAuth\Model\Provider\Engine\Google::getTotp()` and `Magento\TwoFactorAuth\Model\Provider\Engine\Google\TotpFactory::create()`, and as a mock in `Magento\TwoFactorAuth\Test\Unit\Model\Provider\Engine\GoogleTest::setUp()`. None of these seem to need a Magento preference.